crl/updater/updater.go

```diff
@@ -181,31 +183,31 @@ func (cu *crlUpdater) Run(ctx context.Context) error {
 // error at the end.
 func (cu *crlUpdater) Tick(ctx context.Context, atTime time.Time) (err error) {
 	defer func() {
-		// Each return statement in this function assigns its returned value to the
-		// named return variable `err`. This deferred function closes over that
-		// variable, and so can reference and modify it.
+		// This func closes over the named return value `err`, so can reference it.
```

Comment:
Since we only need to _reference_ it, not modify it, we can replace the named return with `var err error` at the top of the function.

Reply:
Explained in https://github.com/letsencrypt/boulder/pull/6382#discussion_r971228125 - it's safer to have the named return so we know it's always assigned to.

Reply:
That's the thing that only works if all of the returns are of the form
```
err = fmt.Errorf("something")
return err
```
Otherwise the err being closed over in the deferred function is separate from the error actually being returned, and the deferred function won't detect that there's been an error. I'm not a fan of requiring every return to be structured that way -- its similar to the known-error-prone
```
wfe.sendError("something")
return
```
in all of the WFE methods, which we [have a bug to remove](https://github.com/letsencrypt/boulder/issues/4980).